README.md

```diff
@@ -1,96 +1,52 @@
+# `rippled` - The XRP Ledger Server
 
-![Ripple](docs/images/ripple.png)
+The XRP Ledger is a decentralized cryptographic ledger powered by a network of peer-to-peer servers. The XRP Ledger uses a novel Byzantine Fault Tolerant consensus algorithm to settle and record transactions in a secure distributed database without a central operator. The server software that powers the XRP Ledger is called `rippled` and is available in this repository under the permissive [ISC open-source license](LICENSE). The XRP Ledger is the home of XRP, a digital asset designed to bridge the many different currencies in use worldwide.
 
-> **Do you work at a digital asset exchange or wallet provider?** 
->
-> Please [contact us](mailto:support@ripple.com). We can help guide your integration.
+The XRP Ledger was created in 2012 with a finite supply of 100 billion units of XRP. Its creators gifted 80 billion XRP to a company, now called [Ripple](https://ripple.com/), to develop the XRP Ledger and its ecosystem. XRP is public, counterparty-less and traded on the open-market, available for anyone to access. Ripple uses XRP as a key piece of its mission of building the Internet of Value: a world in which money moves the way information does today.
 
-Ripple is a network of computers which use the [Ripple consensus algorithm](https://www.youtube.com/watch?v=pj1QVb1vlC0) to atomically settle and record
-transactions on a secure distributed database, the Ripple Consensus Ledger
-(RCL). Because of its distributed nature, the RCL offers transaction immutability
-without a central operator. The RCL contains a built-in currency exchange and its
-path-finding algorithm finds competitive exchange rates across order books
-and currency pairs.
 
-### Key Features
-- **Distributed**
-  - Direct account-to-account settlement with no central operator
-  - Decentralized global market for competitive FX
-- **Secure**
-  - Transactions are cryptographically signed using ECDSA or Ed25519
-  - Multi-signing capabilities
-- **Scalable**
-  - Capacity to process the world’s cross-border payments volume
-  - Easy access to liquidity through a competitive FX marketplace
+## Key Features
 
-## Cross-border payments
-Ripple enables banks to settle cross-border payments in real-time, with
-end-to-end transparency, and at lower costs. Banks can provide liquidity
-for FX themselves or source it from third parties.
+- **[Censorship-Resistant Transaction Processing][]:** No single party decides which XRP transactions succeed or fail, and no one can "roll back" a transaction after it completes. As long as those who choose to participate in the network keep it healthy, they can send and receive XRP in seconds.
+- **[Fast, Efficient Consensus Algorithm][]:** The XRP Ledger's consensus algorithm settles transactions in 4 to 5 seconds, processing at a throughput of up to 1500 transactions per second. These properties put XRP at least an order of magnitude ahead of other top digital assets.
```

Comment:
"processing at a throughput of up to 1500 transactions per second" should probably be: "at rates of over 1500 transactions per second."

Reply:
Might be nice to also cite the article that claims these rates (the environment is a bit more special than the general case that XRPL is running on and it has not yet been independently replicated): http://highscalability.com/blog/2017/10/2/ripple-the-most-demonstrably-scalable-blockchain.html

Reply:
The officially cited stats are that sustained throughput peaked at 1500 tps, thus "up to" and not "over". (It's possible that performance has improved since then, but absent a published test, I don't think it's appropriate to assume.)

As for the link, it's a good one, but I expect this part of the README to change relatively rarely. I'd rather keep to a minimum the number of links to blog articles and similar outside of our control.